go/ct/rlz/conditions.go

```diff
@@ -724,11 +727,26 @@ func (c *isAddressCold) Restrict(generator *gen.StateGenerator) {
 }
 
 func (c *isAddressCold) GetTestValues() []TestValue {
-	return IsAddressWarm(c.key).GetTestValues()
+	property := Property(fmt.Sprintf("warm(%v)", c.key))
```

Comment:
cold

Reply:
As discussed with Herbert in the other conversations, the `property` of `isAddressWarm` and `isAddressCold` has to be the same because it is used to unify constraints on the same property.